cheroot/workers/threadpool.py

```diff
@@ -52,7 +52,6 @@ class WorkerThread(threading.Thread):
     ready = False
     """A simple flag for the calling server to know when this thread
     has begun polling the Queue."""
-
```

Comment:
unrelated + violates the style.

Reply:
Same as above.
Seems like this was either done by precommit hook or flake8 autoformatter used for satisfying precommit hook. Will check again.

Reply:
Added line.